packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -177,7 +177,31 @@ export default ({
     }
   }
 
-  const handleDepositFiatClick = function * () {
+  const handleDepositFiatClick = function * ({
+    payload
+  }: ReturnType<typeof A.handleDepositFiatClick>) {
+    yield put(
+      actions.components.brokerage.showModal(
+        BrokerageModalOriginType.DEPOSIT_BUTTON,
+        'BANK_DEPOSIT_MODAL'
```

Comment:
We use this string on few places, I was thinking to move it to constant and use it like that all around.

Reply:
We should make a separate tech debt ticket for this because 'BANK_DEPOSIT_MODAL' is part of the `ModalNamesType` which has about ~45 different modal names in it. To turn that into an enum and then make the changes everywhere in the app would be a pretty big task!